WcaOnRails/lib/cutoff.rb

```diff
@@ -1,15 +1,20 @@
 # frozen_string_literal: true
 
 class Cutoff
-  attr_accessor :numberOfAttempts, :attemptValue, :event
-  def initialize(numberOfAttempts: nil, attemptValue: nil, event: nil)
-    self.numberOfAttempts = numberOfAttempts
-    self.attemptValue = attemptValue
+  include ActiveModel::Validations
+
+  attr_accessor :number_of_attempts, :attempt_result, :event
+  validates :number_of_attempts, numericality: { only_integer: true }
+  validates :attempt_result, numericality: { only_integer: true }
+
+  def initialize(number_of_attempts: nil, attempt_result: nil, event: nil)
+    self.number_of_attempts = number_of_attempts
+    self.attempt_result = attempt_result
     self.event = event
   end
 
   def to_wcif
-    { "numberOfAttempts" => self.numberOfAttempts, "attemptValue" => self.attemptValue }
+    { "numberOfAttempts" => self.number_of_attempts, "attemptResult" => self.attempt_result }
```

Comment:
I feel like we're going to have these around a lot when it comes to loading/exporting object from/to wcif.
I've handled this through a concern in wc2017's app, see [here](https://github.com/viroulep/wc2017/blob/ff0571040c6222789f5f664e1bc76404cb72dedd/app/models/concerns/wca_model.rb#L28-L35).

The way it works is: you define an `@@obj_info` (like [here](https://github.com/viroulep/wc2017/blob/ff0571040c6222789f5f664e1bc76404cb72dedd/app/models/concerns/wca_model.rb#L28-L35)), and the function do some object manipulation using each `key.underscore` or `key.camelize(:lower)`.
This precise function does not do what you want here (it transforms camelCase attributes in the json to snake_case), but I think the idea can be reused here.
For "simple" object, the `to_wcif` function would just be a call to this concern's function, for more complicated object (like `Competition` or `Registration`), you can first call the "translation" function, and then handle attribute that require a `.to_wcif`.